_docs/kustomize/akashd/deployment.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: akashd
+  name: akash
```

Comment:
Is this the pattern that `gaia` is going with for its "Node" service type? My inclination is for a little bit more explicit information on the Deployment name itself. `akash-node`? We'll need to make it clear between the node and `provider` service. 

Also need to apply the name to labels and selectors.

Reply:
thank you for catching that! I've reverted that change.